dask_cloudprovider/cloudprovider.yaml

```diff
@@ -1,21 +1,32 @@
-cloudprovider: {}
+cloudprovider:
 
-  # ecs: {}
-  #   # scheduler_cpu: 1024  # Millicpu (1024ths of a CPU core)
-  #   # scheduler_mem: 4096  # Memory in MB
-  #   # worker_cpu: 4096  # Millicpu (1024ths of a CPU core)
-  #   # worker_mem: 16384  # Memory in MB
-  #   # n_workers: 2
+  ecs:
+    fargate_scheduler: False  # Use fargate mode for the scheduler
+    fargate_workers: False  # Use fargate mode for the workers
+    scheduler_cpu: 1024  # Millicpu (1024ths of a CPU core)
+    scheduler_mem: 4096  # Memory in MB
```

Comment:
From a Dask perspective it's more natural to me to use text here, like `4096 MB`.  However, I suspect that you've chosen this approach because it's more familiar to AWS users.

Reply:
I chose this because the AWS API requires an integer value in megabytes (apart from the few times it wants a string of the integer ... _grumble, grumble_). If you think it would be more natural to have a string here following the dask approach I could look at parsing it. Are there any utils in distributed for doing this?

Reply:
```python
In [1]: from dask.utils import parse_bytes

In [2]: parse_bytes("4096 MB")
Out[2]: 4096000000

In [3]: parse_bytes?
Signature: parse_bytes(s)
Docstring:
Parse byte string to numbers

>>> parse_bytes('100')
100
>>> parse_bytes('100 MB')
100000000
>>> parse_bytes('100M')
100000000
>>> parse_bytes('5kB')
5000
>>> parse_bytes('5.4 kB')
5400
>>> parse_bytes('1kiB')
1024
>>> parse_bytes('1e6')
1000000
>>> parse_bytes('1e6 kB')
1000000000
>>> parse_bytes('MB')
1000000
>>> parse_bytes('5 foos')  # doctest: +SKIP
ValueError: Could not interpret 'foos' as a byte unit
File:      ~/workspace/dask/dask/utils.py
Type:      function
```

Reply:
See also it's friends, `format_bytes`, `parse_timedelta`, `format_timedelta`

Reply:
Ah nice ok. I'd used `format_bytes` before but didn't know about `parse_bytes`.

Reply:
I'm going to get this merged and have raised #27 to keep track of your comments here.